about.html

```diff
@@ -0,0 +1,137 @@
+            <li class="nav-item"><a href="index.html">Past Events</a></li>
+          </ul>
+          <button class="nav-btn" type="submit">Donate!</button>
+        </nav>
+      </div>
+    </header>
+    <section class="hamburger">
+      <a class="menuIcon" href="#">
+        <svg viewBox="0 0 10 8" width="75">
+          <path d="M1 1h8M1 4h 8M1 7h8" stroke="#585858" stroke-width="1.5" />
+        </svg>
+      </a>
+    </section>
+    <ul class="menu">
+      <li><a class="menu-item closeIcon">X</a></li>
+      <li><a class="menu-item" href="index.html">Home</a></li>
+    </ul>
+
+    <section class="about" id="link-1">
+      <div class="container">
+        <h3 class="about-title">"Hello! Horror Fans"</h3>
+        <h1 class="main-header2">Master Horror Writers Event 2021</h1>
+        <p class="about-paragraph">
+          Events for book fans is an enterprise dedicated to function as a link
+          between the readers and their favourite authors of all the genres and
+          from any country of the world.The company was founded in 2010 by a
+          group of enthusiast book lovers who wanted to have an opportunity to
+          meet their favourite authors.
+        </p>
+        <p class="about-contact">
+          Please contact us per Email for any further questions about Master
+          Horror Writers Event 2021
+        </p>
+        <p class="strong" id="link-5">evensforbookfans@gmail.com</p>
+        <div class="past-events">
+          <h3 class="last-header">
+            Creative horror writers <br />
+            event 2021 Logo
+          </h3>
+        </div>
+        <div class="line"></div>
+        <p class="about-contact">
+          The logo of Event for book fans was decided through the logo
+          competition from 8. June to 7. July.
+        </p>
+
+        <div>
+          <img class="logo-brand" src="images/logo-events.jpg" alt="logo" />
+        </div>
+
+        <div class="past-events">
+          <h3 class="last-header">See past Events</h3>
+          <div class="line"></div>
+
+          <p class="about-contact" id="link-6">
+            Take a look at the last two Events wich took place in San Pablo and
+            New York.
+          </p>
+          <div class="photos-desktop photos-mobile">
+            <div class="contenedor">
+              <img
+                class="event-pictures"
+                src="images/foto2018.jpg"
+                alt="San-Pablo-event"
+              />
+              <div class="centrado sp">SAN PABLO 2018</div>
+            </div>
+            <div class="contenedor">
+              <img
+                class="event-pictures"
+                src="images/foto2020.jpg"
+                alt="New-york-event"
+              />
+              <div class="centrado">NEW YORK 2020</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer class="footer-desk">
```

Comment:
The footer should have a white background on mobile view. The content should be centered vertically and have the same margin from the right and left. KIndly follow the font guidelines from the design.
![footer](https://user-images.githubusercontent.com/35504149/137521776-48df62bf-b3be-48f7-82c3-e028cf77a53a.png)